docs/data-structures/inordertraversal.mdx

```diff
@@ -0,0 +1,206 @@
+---
+title: Inorder traversal of Binary Tree
+slug: /data-structures/inordertraversal
+---
+
+## Introduction
+
+Binary Tree is a type of tree which have atmost two children or we can say atmost degree 2. So to traverse the binary tree according to Depth first search. We have a traversal algorithm called Inorder traversal.
```

Comment:
Few grammatical mistakes over here. Make it:

A binary tree is a type of tree which have at most two children or we can say at most degree 2. So to traverse the binary tree according to Depth-first search. We have a traversal algorithm called Inorder traversal.